File: ueueo-framework/ueueo-blob-storing/src/main/java/Abp/BlobStoring/BlobProviderSaveArgs.cs
using System.IO;
using System.Threading;
using JetBrains.Annotations;

namespace Volo.Abp.BlobStoring;

public class BlobProviderSaveArgs : BlobProviderArgs
{
    [NotNull]
    public Stream BlobStream;//  { get; }

    public boolean OverrideExisting;//  { get; }

    public BlobProviderSaveArgs(
        @Nonnull String containerName,
        @Nonnull BlobContainerConfiguration configuration,
        @Nonnull String blobName,
        @Nonnull Stream blobStream,
        boolean overrideExisting = false,
        CancellationToken cancellationToken = default)
        : base(
            containerName,
            configuration,
            blobName,
            cancellationToken)
    {
        BlobStream = Objects.requireNonNull(blobStream, nameof(blobStream));
        OverrideExisting = overrideExisting;
    }
}

File: ueueo-frameworks/ueueo-blob-storing/src/main/java/com/ueueo/blobstoring/BlobProviderSaveArgs.java
package com.ueueo.blobstoring;

import com.ueueo.threading.CancellationToken;
import org.springframework.lang.NonNull;

import java.io.InputStream;
import java.util.Objects;

public class BlobProviderSaveArgs extends BlobProviderArgs {
    @NonNull
    private InputStream blobStream;

    private boolean overrideExisting;

    public BlobProviderSaveArgs(
            @NonNull String containerName,
            @NonNull BlobContainerConfiguration configuration,
            @NonNull String blobName,
            @NonNull InputStream blobStream,
            boolean overrideExisting,
            CancellationToken cancellationToken) {
        super(
                containerName,
                configuration,
                blobName,
                cancellationToken);
        this.blobStream = Objects.requireNonNull(blobStream);
        this.overrideExisting = overrideExisting;
    }

    @NonNull
    public InputStream getBlobStream() {
        return blobStream;
    }

    public boolean isOverrideExisting() {
        return overrideExisting;
    }
}
